hermes/probe/read/read_test.go

```diff
@@ -65,16 +78,44 @@ func TestReadFile(t *testing.T) {
 		TotalSpaceAllocatedMib: int64(100),
 		BucketName:             "test_bucket_probe0",
 	}
+
 	var err error
 	if target.LatencyMetrics, err = metrics.NewMetrics(hp, probeTarget); err != nil {
-		t.Fatalf("Metric set up failed due to %s", err.Error())
+		t.Fatalf("metrics.NewMetrics(): %v", err)
 	}
-	fileID := int32(6)
-	fileSize := 50
-	if err := create.CreateFile(ctx, target, fileID, fileSize, client, nil); err != nil {
+
+	ctx := context.Background()
+	client := fakegcs.NewClient()
+	bucketName := "test_bucket_probe0"
+	fakeBucketHandle := client.Bucket(bucketName)
+	if err := fakeBucketHandle.Create(ctx, bucketName, nil); err != nil {
 		t.Error(err)
```

Comment:
This should be Fatalf, as it's a set-up error.

Reply:
changed to Fatalf